nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/processors/SNMPUtils.java

```diff
@@ -48,77 +42,58 @@
 /**
  * Utility helper class that simplifies interactions with target SNMP API and NIFI API.
  */
-abstract class SNMPUtils {
+public final class SNMPUtils {
```

Comment:
Probably the right choice for this... but see my "public API comments" from before. If someone was subclassing this class...